content/main.js

```diff
@@ -390,16 +390,24 @@ var tblatex = {
       div.parentNode.removeChild(div);
   }
 
-  function replace(string, pattern, replacement) {
+  function replace_marker(string, replacement) {
+    var marker = "__REPLACE_ME__";
+    var oldmarker = "__REPLACEME__";
+    var l = marker.length;
```

Comment:
actually where is `l` used?

Reply:
if it's not used we should just remove `l`

Reply:
The variable 'l' is used in line `var p2 = string.substring(i+l);` ... Maybe `l` is not the best name for a variable (is it `l` or `1`?), so I changed it to `len`.